pkg/cmd/experimental/syncgroups/groupdetector/groupdetector.go

```diff
@@ -26,6 +26,10 @@ func (l *GroupBasedDetector) Exists(ldapGroupUID string) (bool, error) {
 		return false, err
 	}
 
+	if group == nil {
+		return false, nil
```

Comment:
Actually, never mind all of that. I can't read. The search request is:
```
&{cn=group1,ou=groups,ou=adextended,dc=example,dc=com 0 0 0 0 false (objectClass=*) [cn dn] []}
```
which translates to
```
&SearchRequest{
    BaseDN:       "cn=group1,ou=groups,ou=adextended,dc=example,dc=com",
    Scope:        ScopeBaseObject,
    DerefAliases: NeverDerefAliases,
    SizeLimit:    None,
    TimeLimit:    None,
    TypesOnly:    false,
    Filter:       "(objectClass=*)",
    Attributes:   ["cn", "dn"],
    Controls:     none,
}
```

I think everything is working as expected now.